src/core/parser.js

```diff
@@ -416,10 +416,6 @@ var Parser = (function ParserClosure() {
 
       return imageStream;
     },
-    fetchIfRef: function Parser_fetchIfRef(obj) {
-      // not relying on the xref.fetchIfRef -- xref might not be set
```

Comment:
Also, note how this comment doesn't actually agree entirely with the code. Reading it, I'd expect the next line to read e.g. like this instead:
```javascript
return (this.xref && isRef(obj) ? this.xref.fetch(obj) : obj);
```